fsspec/tests/test_mapping.py

```diff
@@ -72,7 +72,7 @@ def test_pickle():
 
 
 def test_keys_view():
-    # https://github.com/intake/filesystem_spec/issues/186
+    # https://github.com/fsspec/fsspec/issues/186
```

Comment:
I guess this one is wrong

Reply:
```suggestion
    # https://github.com/fsspec/filesystem_spec/issues/186
```

Something to consider :) (since the package is already named fsspec on pypi)